applications/salk-portal/src/components/ExperimentSidebar.jsx

```diff
@@ -45,6 +45,27 @@ const useStyles = makeStyles({
       },
     },
 
+    '& .population-label-container': {
+      display:'flex',
+      flexDirection:'row',
+      alignContent:'center',
+      gap:'.5rem'
+    }, 
+
+    '& .population-label-text': {
+      lineHeight: '0.4rem',
+    },
+
+
+    '& .population-label-dot': {
```

Comment:
@paularah if you notice in figma there is a border around the dot? did u miss it or didn't know how to add it?